rd_ui/app/scripts/ng_highchart.js

```diff
@@ -325,6 +325,36 @@
                 series = seriesCopy;
               }
 
+              if (chartOptions['sortY'] === true || chartOptions['sortY'] === undefined) {
+                var sortable = true;
+
+                for (var i = 0; i < series.length; i++) {
+                  if (series[i].data.length != 1) {
```

Comment:
Great suggestions. I forgot about Underscore; it's been a while since I've used it. I've updated and retested.